app/validators/file_upload_validator.rb

```diff
@@ -0,0 +1,57 @@
+class FileUploadValidator < ActiveModel::Validator
+  MIN_FILE_SIZE = 5 # KB
+  MAX_FILE_SIZE = 20 # MB
+
+  ALLOWED_CONTENT_TYPES = %w[
+    application/msword
+    application/pdf
+    application/rtf
+    application/vnd.ms-excel
+    application/vnd.oasis.opendocument.text
+    application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
+    application/vnd.openxmlformats-officedocument.wordprocessingml.document
+    image/bmp
+    image/gif
```

Comment:
This also not allowed according to the copy (and Civil don't allow it)

Reply:
I have no idea where this list came from because gifs is hilarious 😆